File: fizz_buzz.py
for num in range(1,101):
    if num % 3==0 and num % 5==0:
        state="FizzBuzz"
    elif num % 3 == 0:
        state="Fizz"
    elif num  % 5 == 0:
        state="Buzz"
    else:
        state=""
    print(f"{num}  {state}")

File: fizzbuzz.js
for (let i = 1; i <= 100; i++) {
  if (i % 5 == 0 && i % 3 == 0) {
    state = i + 'fizzBuzz';
  }else if (i % 3 == 0) {
    state = i + 'fizz';
  }else if (i % 5 == 0) {
    state = i + 'buzz';
  }else {
    state = i;
  }

  console.log(state);
};
